great_expectations/cli/suite.py

```diff
@@ -303,7 +298,7 @@ def _suite_new_workflow(
         raise e
 
 
-def _suite_new_convert_flags_to_interactive_mode(
+def _suite_convert_flags_to_interactive_mode(
```

Comment:
@cdkini Does the method name change imply that it will be used not only for `SUITE NEW`?